gpd/gpd-pocket/default.nix

```diff
@@ -0,0 +1,87 @@
+{ lib, pkgs, ... }:
+
+# GPD Pocket support
+#
+# Most of the configuration comes from the Arch Linux Wiki:
+# https://wiki.archlinux.org/index.php/GPD_Pocket
+#
+# Missing:
+# - media keys
+# - brightness control
+# - video over USB-C
+
+{
+  imports = [
+    ../../common/cpu/intel
+    ../../common/pc/laptop
+  ];
+
+  # kernel
+  # TODO: use https://github.com/jwrdegoede/linux-sunxi?
+  boot = {
+    kernelParams = ["fbcon=rotate:1"];
+    kernelPackages = pkgs.linuxPackages_4_14;
+  };
+
+  # wifi
+  nixpkgs.config.allowUnfree = true;
```

Comment:
It should be up to the end user whether to allow unfree packages or not, even when crucial to hardware functionality.